docs_src/microservices/support/logging/Ch-Logging.md

```diff
@@ -60,44 +60,15 @@ by EdgeX Foundry Logging Service: file or MongoDB.
 
 ## Configuration Properties
 
+The following configuration properties are specific to the support-logging service. Please refer to the general Configuration documentation for configuration properties common across all services.
   
   |Configuration  |     Default Value     |             Dependencies|
   | --- | --- | -- |
 | **Entries in the Writable section of the configuration can be changed on the fly while the service is running if the service is running with the `–registry / -r` flag** |
```

Comment:
Change `–registry / -r` to `–configProvider/ -cp` or just use same text used for Core &  Metadata